benches/containerd-shim-benchmarks/benches/wasi-demo-app-benchmarks.rs

```diff
@@ -0,0 +1,63 @@
+use std::process::Command;
+use std::time::{Duration, Instant};
+
+use criterion::{criterion_group, criterion_main, Criterion};
+
+static RUNTIMES: &[&str] = &["wasmtime", "wasmedge", "wasmer", "wamr"];
+
+fn run_container(runtime: &str, image_base_name: &str) -> Duration {
+    let start = Instant::now();
+
+    let output = Command::new("sudo")
+        .args([
+            "ctr",
+            "run",
+            "--rm",
+            &format!("--runtime=io.containerd.{}.v1", runtime),
+            &format!("ghcr.io/containerd/runwasi/{}:latest", image_base_name),
+            "testwasm",
+            &format!("{}.wasm", image_base_name),
+            "echo",
+            "hello",
+        ])
+        .output()
+        .expect("Failed to execute command");
+
+    if !output.status.success() {
+        panic!(
+            "Container failed to run: {}",
+            String::from_utf8_lossy(&output.stderr)
+        );
+    } else {
+        let stdout_str = String::from_utf8_lossy(&output.stdout);
+        assert!(stdout_str.contains("hello"));
+    }
+
+    start.elapsed()
+}
+
+fn benchmark_startup(c: &mut Criterion) {
```

Comment:
I don't know much about criterion, can you pass variables here? If so it might be nice to forgo the double `for` loops and pass the name of the image you want to run making this whole thing re-useable.

Reply:
I've done a refactor of this file. Please take a look again!